tests/pytests/unit/states/test_pip.py

```diff
@@ -67,5 +67,10 @@ def test_issue_64169(caplog):
 
         # Confirm that the state continued to install the package as expected.
         # Only check the 'pkgs' parameter of pip.install
-        mock_install_call_args, mock_install_call_kwargs = mock_pip_install.call_args
+        # pylint: disable=unpacking-non-sequence
+        (
+            mock_install_call_args,
```

Comment:
This will be fixed in this PR https://github.com/saltstack/salt/pull/64337
Can you remove this pylint exception and we will rebase when that PR is merged in